hibernate-reactive-core/src/test/java/org/hibernate/reactive/AutoincrementTest.java

```diff
@@ -178,21 +171,17 @@ public int hashCode() {
 		Double nullDouble;
 		Byte nullByte;
 
-		BigDecimal decimal;
```

Comment:
I removed BigDecimal and BigInteger from this test because DB2 does not support these types yet, and testing the different data types was not the main point of the test (the point of the test is auto incrementing) and we have `BasicTypesAndCallbacksTest` which does cover these types